Lib/test/test_trace.py

```diff
@@ -412,6 +413,14 @@ def test_issue9936(self):
         self.assertIn(modname, coverage)
         self.assertEqual(coverage[modname], (5, 100))
 
+    def test_coverageresults_update(self):
+        outfile = TESTFN + '-outfile'
+        with open(outfile, 'wb') as f:
+            pickle.dump(({}, {}, {'caller': 1}), f, protocol=1)
+        self.addCleanup(unlink, outfile)
+        results = trace.CoverageResults({}, {}, outfile, {})
+        self.assertEqual(results.callers, {'caller': 1})
```

Comment:
```suggestion
    def test_coverageresults_update(self):
        # Update empty CoverageResults with a non-empty infile.
        infile = TESTFN + '-infile'
        with open(infile, 'wb') as f:
            pickle.dump(({}, {}, {'caller': 1}), f, protocol=1)
        self.addCleanup(unlink, infile)
        results = trace.CoverageResults({}, {}, infile, {})
        self.assertEqual(results.callers, {'caller': 1})
```
Apply this when add blurb, which I believe will trigger CI rerun anyway.